internal/clients/http/gcp/gcp_client.go

```diff
@@ -156,6 +204,11 @@ func (c *gcpClient) InsertInstances(ctx context.Context, params *clients.GCPInst
 		},
 	}
 
+	if params.LaunchTemplateName != "" {
+		template := fmt.Sprintf("global/instanceTemplates/%s", params.LaunchTemplateName)
```

Comment:
I wonder if there is a possibility to have a different prefix than `global/instanceTemplate` but I think it would be pretty easy to differentiate later using string prefix comparison so this is fine.